### PR TITLE
Doc 4441 - Fixed various links

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
Fixes of 4.9 topics which were linking to 4.8 topics and also linking to wrong topics (community going to enterprise and vice versa).

### CODE REVIEW

File: content/en/platform/corda/4.9/community/blob-inspector.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ title: Blob Inspector
 
 # Blob inspector
 
-The Corda blob inspector tool gives you a human-readable view of content stored in a [custom binary serialization format](../../../../../en/platform/corda/4.8/open-source/serialization.md).
+The Corda blob inspector tool gives you a human-readable view of content stored in a [custom binary serialization format](serialization.md).
```
Comment (yarivus) [inline content/en/platform/corda/4.9/community/blob-inspector.md:21]: Should be .html?!

File: content/en/platform/corda/4.9/community/blob-inspector.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ This property is materialized into `NodeInfo` and is output under the `deseriali
 
 ## Classpath
 
-If you run the blob inspector without any `.jar` files on the classpath, then it will deserialize objects using the class carpenter, (see [Object serialization](../../../../../en/platform/corda/4.8/open-source/serialization.md)).
+If you run the blob inspector without any `.jar` files on the classpath, then it will deserialize objects using the class carpenter, (see [Object serialization](serialization.md)).
```
Comment (yarivus) [inline content/en/platform/corda/4.9/community/blob-inspector.md:104]: Same here... .html?!

File: content/en/platform/corda/4.9/community/secure-coding-guidelines.md [hunk 3]
```diff
@@ -62,5 +62,5 @@ Make sure your contracts are secure. Check that:
 
 ## Related Content
 Learn more about:
-* [Writing flows](../../../../../en/platform/corda/4.8/enterprise/flow-state-machines.md)
-* [Contracts](../../../../../../en/platform/corda/4.8/enterprise/cordapps/api-contracts.html)
+* [Writing flows](../enterprise/flow-state-machines.md)
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/community/secure-coding-guidelines.md:65]: presume this is deliberately still pointing to enterprise?

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.9/community/shell.md [hunk 4]
```diff
@@ -479,7 +479,7 @@ Use the different flow commands available to make changes on the ledger. You can
 
 #### Query flow data
 
-The shell can be used to query flow data. For more information on the types of data that can be queried and instructions for doing so, see the documentation on [Querying flow data](../../4.8/enterprise/node/operating/querying-flow-data.html#querying-flow-data-via-the-node-shell).
+The shell can be used to query flow data. For more information on the types of data that can be queried and instructions for doing so, see the documentation on [Querying flow data](../enterprise/node/operating/querying-flow-data.html#querying-flow-data-via-the-node-shell).
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/community/shell.md:482]: presume this is deliberately still pointing to enterprise?

File: content/en/platform/corda/4.9/community/tutorial-cordapp.md [hunk 5]
```diff
@@ -270,7 +270,7 @@ You can see the newly-created IOU by running `run vaultQuery contractStateType:
 
 ### Via the h2 web console
 
-You can connect directly to your node’s database to see its stored states, transactions, and attachments. Follow the instructions in [Node database](../../4.8/enterprise/node/operating/node-database.md).
+You can connect directly to your node’s database to see its stored states, transactions, and attachments. Follow the instructions in [Node database](../enterprise/node/operating/node-database.md).
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/community/tutorial-cordapp.md:273]: presume this is deliberately still pointing to enterprise?

File: content/en/platform/corda/4.9/enterprise/cordapps/debugging-a-cordapp.md [hunk 6]
```diff
@@ -38,7 +38,7 @@ running via the node driver to debug your CorDapp. You can debug the CorDapp:
 
 ### Debug with the nodes in-process
 
-1. [Define a network using the node driver](../../../../../../en/tutorials/corda/4.9/os/supplementary-tutorials/tutorial-integration-testing.md).
+1. [Define a network using the node driver](../../../../../tutorials/corda/4.9/community/supplementary-tutorials/tutorial-integration-testing.md).
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/enterprise/cordapps/debugging-a-cordapp.md:41]: deliberately community, I presume?

**Review by nadinequinr3 — APPROVED** [review]
Noted a few links from enterprise to community (and vice versa) but I'm guessing they're deliberate and didn't comment on them all... Otherwise, LGTM

File: content/en/platform/corda/4.9/community/blob-inspector.md (hunk 1 above)
Comment (nadinequinr3) [reply content/en/platform/corda/4.9/community/blob-inspector.md:21]: I think this is up in the air a bit.. I think @colin-hillis suspects that a new platform would expect files to use the file extension before build so we weren't to waste time switching existing `.md` paths..

Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/community/blob-inspector.md:21]: Yup based on what Colin said I'll leave existing .md links as .md for now.

File: content/en/platform/corda/4.9/community/shell.md (hunk 4 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/community/shell.md:482]: Yup, there's no community version in that case.

File: content/en/platform/corda/4.9/enterprise/cordapps/debugging-a-cordapp.md (hunk 6 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/enterprise/cordapps/debugging-a-cordapp.md:41]: Yup, the original hadn't been updated from the old "os" format.

File: content/en/platform/corda/4.9/community/tutorial-cordapp.md (hunk 5 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/community/tutorial-cordapp.md:273]: Yup in that case there was no community equivalent.

File: content/en/platform/corda/4.9/community/secure-coding-guidelines.md (hunk 3 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/community/secure-coding-guidelines.md:65]: Yeah, no community equivalent.

File: content/en/platform/corda/4.9/community/blob-inspector.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/community/blob-inspector.md:104]: Hi Yariv, based on what @colin-hillis  said about converting in future to Antora, he thought it based to leave .md links as .md for now.

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**